fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -3958,6 +3959,9 @@ public void renameColumn(Database db, Table table, ColumnRenameClause renameClau
         if (!(table instanceof OlapTable)) {
             throw new DdlException("Column rename now only supports olap table.");
         }
+        if (db.getFullName().equals(StatsConstants.STATISTICS_DB_NAME)) {
```

Comment:
whether need to put `_statistic_` as a system db, so the check of `isSystemDatabase` can be applied here?

Reply:
`_statistic_` can not be added to `isSystemDatabase`, otherwise its tables will be ignored in some programs, such as Tablet Scheduler.

Reply:
`_statistic_` is different with `sys`, `information_schema`, data in `_statistic_` is stored on be as OLAP table format.

Reply:
If there was another database identical to `_statistic_`, we could add an api to unify it, but that's not needed now.